lib/zone.ts

```diff
@@ -1240,7 +1240,7 @@ const Zone: ZoneType = (function(global: any) {
 
     public toString() {
       if (this.data && typeof this.data.handleId !== 'undefined') {
-        return this.data.handleId;
+        return Object.prototype.toString.call(this.data.handleId);
```

Comment:
I don't think this will have the expected behavior, as it will always return `[object Number]`  (which would cause #341 again).
I think it should be `this.data.handleId.toString()` here.

Reply:
Yeah, you are right, thanks!